exercises/practice/mazy-mice/src/test/java/MazeGeneratorTest.java

```diff
@@ -0,0 +1,193 @@
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Ignore;
+
+import java.util.Arrays;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class MazeGeneratorTest {
+    private static final char EMPTY_CELL = ' ';
+    private static final Set<Character> ALLOWED_SYMBOLS = Set.of(
+            EMPTY_CELL, // space
+            '┌', // box drawings light down and right
+            '─', // box drawings light horizontal
+            '┬', // box drawings light down and horizontal
+            '┐', // box drawings light down and left
+            '│', // box drawings light vertical
+            '└', // box drawings light up and right
+            '┴', // box drawings light up and horizontal
+            '┘', // box drawings light up and left
+            '├', // box drawings light vertical and right
+            '┤', // box drawings light vertical and left
+            '┼', // box drawings light vertical and horizontal
+            '⇨'  // rightwards white arrow
+    );
+    private static final char VISITED_CELL = '.';
+    private static final Dimensions RECTANGLE = new Dimensions(6, 18);
+    private MazeGenerator sut;
+
+    @Before
+    void SetUp() {
+        sut = new MazeGenerator();
+    }
+
+    @Test
+    void mazeIsNotNull() {
+        var maze = sut.generatePerfectMaze(RECTANGLE);
+
+        assertThat(maze)
+                .as("The maze is not null.")
+                .isNotNull();
+
+        Arrays.stream(maze)
+                .map(String::new)
+                .forEach(System.out::println);
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    void the_dimensions_are_correct() {
```

Comment:
> Please use a single style for naming test methods. Since the other test methods use camelCase, I'd suggest changing this one to `theDimensionsAreCorrect`.

The test method name 'the_dimensions_are_correct' in MazeGeneratorTest has been refactored to 'theDimensionsAreCorrect'. This is in line with Java's camelCase naming convention, improving readability and maintaining a consistent code style.